services/api/src/models/user.js

```diff
@@ -21,8 +21,8 @@ schema.methods.removeAuthToken = function (jti) {
   this.authInfo = this.authInfo.filter((token) => token.jti !== jti);
 };
 
-schema.methods.createAuthToken = function ({ ip, userAgent, country }) {
-  const { token, payload } = createAuthToken(this.id);
+schema.methods.createAuthToken = function ({ ip, userAgent, country }, tokenOption = {}) {
```

Comment:
So the last argument is a bit special ... its a not required ... expect for when using the login as user.
I split them apart because the use of these goes to different places 

```
createAuthToken(authInfo, tokenFields)
```

I think it make sense to keep as is, instead of trying to merge into one object and then figure out which field goes to what function

Reply:
I'll leave it up to you but as a consumer of that method I would find it quite confusing.

Reply:
I don't care either way. I just think we at some point want to clean up what we stick in the JWT tokens.

I don't know who came up with `sub`, `kid` and `jti` but it's awful

Reply:
It's [JWT lingo](https://stackoverflow.com/questions/43867440/whats-the-meaning-of-the-kid-claim-in-a-jwt-token) so kind of makes sense, but agree it's hard to parse. Would like to at least prevent that from making it's way into the db if possible.